numcosmo/math/ncm_stats_dist_nd_kde_studentt.c

```diff
@@ -27,12 +27,40 @@
 /**
  * SECTION:ncm_stats_dist_nd_kde_studentt
  * @title: NcmStatsDistNdKDEStudentt
- * @short_description: An N dimensional probability distributions using studenttian KDE
+ * @short_description: An N dimensional probability distributions using Multivariate Student's t KDE
  *
- * An arbitrary N dimensional probability distribution using studenttian KDE.
+ * This object provides the tools to perform a radial basis interpolation
+ * in a multidimensional function, using a multivariate Student's t distribution.
+ * The goal is to use a polinomial function generated by the interpolation,
+ * such that we are able to sample the original functio from this new interpolation function. 
+ * For more informations about radial basis interpolation,
+ * check [https://core.ac.uk/download/pdf/37320748.pdf].
+ * A brief description of the radial basis interpolation method can be found below.
+ *
+ * Given a d-simensional function $g(x): \mathbf{R}^d \rightarrow \mathbf{R}$, a radial basis  
+ * function $\phi(x)$ is used such that $s(x) = \sum_i^n \lambda \phi(|x-xi|), x \in \mathbf{R}$.
+ * The matrix lambda is found such that s(x_i) = g(x_i), being $x_i$ the sample points
+ * and the points $\phi(|x-x_i|)$ generate a symmetric $n \times n$ matrix using the radial basis function,
+ * in this case the Multivariate Student's t function. Once the Lambda matrix is found, 
+ * one may use $s(x)$ to sample values from $g(x)$, which is easier to do since $s(x)$ is 
+ * a polinomial function.
+ *
+ * We $s(x)$ to be a probability function so we can sample from it. Therefore the Lambda matrix containing the
+ * weights is seen as the probability density of our probability function and it must be minimized
+ * such that its values are always positive and sum up to one. 
  * 
+ * We use the Multivariate Student's t probability density function as the radial basis function, which is given by:
+ * $\phi(x) = \frac{\Gamma[(\nu+p) / 2]}{\Gamma(\nu / 2) \nu^{p / 2} \pi^{p / 2}|\mathbf{\Sigma}|^{1 / 2}}\left[1+\frac{1}{\nu}
+ * (\mathbf{x}-\boldsymbol{\mu})^{T} \boldsymbol{\Sigma}^{-1}(\mathbf{x}-\boldsymbol{\mu})\right]^{-(\nu+p) / 2}$,
+ * where $\nu$ is the number of degrees of freedom, $p$ is the dimension, $\Sigma$ is the covariant matrix,
+ * $\mu$ is the mean (which is only defined if $\nu > 1$) and $x$ are the points to be evaluated.
+ *
+ * The user must provide the following input values:$p$ - ncm_stats_dist_nd_kde_studentt_new,
+ * $LOOCV$ - ncm_stats_dist_nd_kde_studentt_new, $\nu$ - ncm_stats_dist_nd_kde_studentt_new,
+ * $vec{x}$ ncm_stats_dist_nd_kde_studentt_prepare_interp.  
+ *
```

Comment:
I've written a short description of the radial basis interpolation method, the goal of the algorithm, and the minimum input from the user. I did not describe other inputs since these are the only absolutely necessary. The other function may or may not be used if wanted by the user.

Let me know if you feel like this isn't complete enough and if I should write a better description of the algorithm or of the method.